### ISSUE TITLE
Does not recognize forwardRef

### ISSUE THREAD

**Issue by eps1lon** [merceyz/typescript-to-proptypes#2]
Looks like it does not add propTypes to forwardRef components. I can provide a full repro if you need one.

See https://github.com/mui-org/material-ui/pull/17841/commits/6e67426ba68cd3c245a848e861e28af4fa424f09

**Comment by merceyz** [comment]
Turns out I wasn't watching this repository, so sorry about the slow response.

---

> Looks like it does not add propTypes to forwardRef components

This is documented in the readme, because at the time, material-ui didn't need it, so I didn't look into it.

---

I looked into it now and there seems to be no difference between `React.memo` and `React.forwardRef` when it comes to getting the type of the props, so i'll fix that right away.
